src/models/linkedCells/LinkedCells.cpp

```diff
@@ -0,0 +1,49 @@
+//
+// Created by daniel on 22.05.24.
+//
+
+#include "LinkedCells.h"
+
+LinkedCells::LinkedCells(Force &force, double deltaT, std::array<double, 3> domainSize,
+                         double rCutOff, double sigma, FileHandler::inputFormat inputFormat,
+                         FileHandler::outputFormat outputFormat,
+                         std::array<std::pair<Side, enumsStructs::BoundaryCondition>, 6> &
+                         boundarySettings) : Model(particles, force, deltaT, inputFormat,
+                                                  outputFormat),
+                                            particles(domainSize, rCutOff),
+                                            boundarySettings{boundarySettings} {
+    threshold = pow(2.0, 1.0 / 6.0) * sigma;
+}
+
+void LinkedCells::processBoundaries() {
+    for (auto setting: boundarySettings) {
```

Comment:
```suggestion
    for (auto &[side, condition] : boundarySettings) {
```
FYI, you can give both pair members a name, making the code slightly nicer to read imo.